doc/sphinx-guides/source/installation/installation-main.rst

```diff
@@ -21,7 +24,12 @@ Execute the installer script like this::
         # cd dvinstall
         # ./install
 
-The script will prompt you for some configuration values. If this is a test/evaluation installation, it should be safe to accept the defaults for most of the settings:
+**NEW in Dataverse 4.3:** It is no longer necessary to run the installer as root!
```

Comment:
Nice! At some point I should reconfigure Vagrant to run the installer as non-root now that this is supported. #1934 and #2443 are highly related to the fact that previously the installer had to be run as root.